src/ui/TemplateMessageItemBody/index.tsx

```diff
@@ -165,16 +162,16 @@ export function TemplateMessageItemBody({
         }
         if (
           templateData.view_variables
-          || parsedUiTemplate[0].type === 'carouselView'
+          || parsedUiTemplate[0].type === CarouselType
           || typeof parsedUiTemplate[0]['items'] === 'string'
           || parsedUiTemplate[0]['spacing']
         ) {
           if (!templateData.view_variables) {
             logger.error('TemplateMessageItemBody | template key suggests composite template but template data is missing view_variables: ', templateKey, templateData);
             throw new Error();
           }
-          const carouselItem: CarouselItem = parsedUiTemplate[0] as CarouselItem;
-          if (carouselItem.type !== 'carouselView'
+          const carouselItem: CarouselItem = parsedUiTemplate[0] as unknown as CarouselItem;
```

Comment:
```suggestion
          const carouselItem = parsedUiTemplate[0] as unknown as CarouselItem;
```
I guess it should be okay without the first one?